### PR TITLE
[CCTRI-3275] - implement health endpoint

### PR DESCRIPTION
**Pull request by mstoro**

### CODE REVIEW

File: code/api/utils.py [hunk 1]
```diff
@@ -122,3 +124,57 @@ def jsonify_data(data):
...
+        },
+    }
+
+
+class BearerAuth(requests.auth.AuthBase):
+    def __init__(self, token):
+        self.token = token
+
+    def __call__(self, request_instance):
+        request_instance.headers['authorization'] = f'Bearer {self.token}'
```
Comment (oshynk) [inline code/api/utils.py:179]: Shouldn't this header be started with upper case?

File: code/api/client.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import requests
+from flask import current_app
+from requests.exceptions import ConnectionError, SSLError
+from http import HTTPStatus
```
Comment (oshynk) [inline code/api/client.py:4]: Seems import order is wrong here and in dashboard.py 🙃

File: code/api/utils.py (hunk 1 above)
Comment (mstoro) [reply code/api/utils.py]: Yeah, thanks. https://github.com/CiscoSecurity/tr-05-serverless-logrhythm/pull/2/commits/3450b6dbb790f991053231807e25432e749d07c8

File: code/api/client.py (hunk 2 above)
Comment (mstoro) [reply code/api/client.py]: https://github.com/CiscoSecurity/tr-05-serverless-logrhythm/pull/2/commits/3450b6dbb790f991053231807e25432e749d07c8

**Review by oshynk — APPROVED** [review]

File: code/config.py [hunk 3]
```diff
@@ -3,4 +3,11 @@
...
     settings = json.load(open('container_settings.json', 'r'))
-    VERSION = settings["VERSION"]
+    VERSION = settings['VERSION']
+    USER_AGENT = ('SecureX Threat Response Integrations '
+                  '<tr-integrations-support@cisco.com>')
+    CTR_DEFAULT_ENTITIES_LIMIT = 100
+
+    # TODO replace http with https before release
+    LOGRHYTHM_API_ENDPOINT = 'http://{host}/lr-search-api/actions'
+    HEALTH_IP = '192.0.2.1'
```
Comment (oshynk) [inline code/config.py:13]: Forgot to ask. Why do you need this IP explicit here?

Comment (mstoro) [reply code/config.py:13]: According to use case we should make search for that IP. https://github.com/threatgrid/integrations/blob/master/LogRhythm/Use_Case_LogRhythm.md#health

**Review by TarasMaliuchenko — APPROVED** [review]

**Pull request merged**